### PR TITLE
Create doc_request.md, an issue template for documentation updates

### PR DESCRIPTION
**Pull request by nenaddedic**

### CODE REVIEW

File: CONTRIBUTING_DOCS.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+You generally only need to submit a CLA once, so if you've already submitted one
+(even if it was for a different project), you probably don't need to do it
+again.
+
+## Updating Documentation
+
+For a small update (examples: fixing a typo, updating a dead link), just create a PR and have it reviewed.
+
+For a larger update (examples: documenting a new feature, rewriting an entire page for better comprehension),
+file a [https://github.com/grafeas/kritis/issues/new?template=doc_request.md](documentation request issue),
```
Comment (aysylu) [inline CONTRIBUTING_DOCS.md:24]: the other way around: [text](link)

File: CONTRIBUTING_DOCS.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+and as you create PRs, make sure to associate them to this issue.
+
+## Requesting an Update
+
+Please file a [https://github.com/grafeas/kritis/issues/new?template=doc_request.md](documentation request issue)
```
Comment (aysylu) [inline CONTRIBUTING_DOCS.md:29]: ditto here for links

File: CONTRIBUTING_DOCS.md (hunk 1 above)
Comment (nenaddedic) [reply CONTRIBUTING_DOCS.md]: Thanks!

File: CONTRIBUTING_DOCS.md (hunk 2 above)
Comment (nenaddedic) [reply CONTRIBUTING_DOCS.md]: Yup, thanks!

**Pull request merged**